cfbs/utils.py

```diff
@@ -295,7 +294,6 @@ def immediate_files(path):
 
 
 def path_append(dir, subdir):
-    dir = os.path.abspath(os.path.expanduser(dir))
```

Comment:
Here, your commit message should include more information on why. It looks like you're changing the implementation just to make it nicer to test. It could be useful that this function always "normalizes" paths like this. I see that it's only used once in this codebase, so probably doesn't matter too much, but it's also in cf-remote (copy-pasted) where it's used a bit more.